src/utils/builder.js

```diff
@@ -527,8 +531,12 @@ const build = (filename, value) => {
       }
 
       // Parse XML data
-      const options = getOptions(filename);
-      parseStringPromise(value, options).then(result => {
+      const dom = (new DOMParser()).parseFromString(value, "text/xml");
+      const result = JSON.parse(xml2json(dom, ''));
```

Comment:
Maybe `xml2json` could have a `parseFromString` where you pass only the `value` as parameter and all DOM creation and JSON parse is handled in there. In the means this could be replaced by a single
js```
import { parseFromString } from './data/xml2json';
...
      const result = parseFromString(value);
```

Reply:
Good idea, will do that.